tests/helpers.py

```diff
@@ -320,14 +318,12 @@ def plot(b_csvs, g_csvs, g_x, g_y, b_x, b_y, trials, seeds, plt_file, env_id,
         df_g = pd.read_csv(g_csvs[trial])
         df_b = pd.read_csv(b_csvs[trial])
 
-        plt.plot(
-            df_g[g_x],
-            df_g[g_y],
-            label='garage_trial%d_seed%d' % (trial + 1, seed))
-        plt.plot(
-            df_b[b_x],
-            df_b[b_y],
-            label='baselines_trial%d_seed%d' % (trial + 1, seed))
+        plt.plot(df_g[g_x],
+                 df_g[g_y],
+                 label='garage_trial%d_seed%d' % (trial + 1, seed))
+        plt.plot(df_b[b_x],
+                 df_b[b_y],
+                 label='baselines_trial%d_seed%d' % (trial + 1, seed))
```

Comment:
please replace 'baselines' with rlkit, or the name of the repo you're actually running in this benchmark.